app/controllers/repos_controller.rb

```diff
@@ -47,13 +47,12 @@ def edit
   end
 
   def update
-    updated = Repo.find(params[:id])
-    if updated.update_attributes(repo_params)
-      updated.cohorts.build
+    @repo = Repo.find(params[:id])
```

Comment:
As with the `StudentsController#update` we're going to throw an exception here if `params[:id]` does not refer to a `Repo` that can be found.

Note to @Hamled: We should deal with that case as well.